test/integration.js

```diff
@@ -0,0 +1,146 @@
+    const url = baseURL;
+    const expectedTitle = 'View Source Conference 2016';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.getTitle().then(function(title) {
+                assert.equal(title, expectedTitle)
+            });
+            done();
+        });
+    });
+
+    test.it('should have a footer & footer should have a link', function(done) {
+        const selector = 'footer ul .footer_social_twitter a';
+        const expectedText = '@ViewSourceConf';
+
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.findElement(webdriver.By.css(selector)).then(function(link) {
+                link.getText().then(function(text) {
+                    assert.equal(text.toLowerCase(), expectedText.toLowerCase());
+                });
+            });
+            done();
+        });
+    });
+});
+
+test.describe('The conference pages...', function() {
+    this.timeout(timeOut);
+
+    ['seattle-2016', 'berlin-2016'].forEach(function(conference) {
+        const url = `${baseURL}/${conference}`;
+
+        test.it(`should have a ${conference} register link`, function(done) {
+            const registerSelector = '.hero a[href*="ti.to"]';
+            const expectedText = 'Register';
+
+            const driver = new webdriver.Builder()
+                .forBrowser('firefox')
+                .build();
+
+            driver.get(url).then(function() {
+                driver.findElement(webdriver.By.css(registerSelector)).then(function(link) {
+                    link.getText().then(function(text) {
+                        assert.equal(text, expectedText);
+                    });
+                });
+                done();
+            });
+        });
+
+        test.it(`should have a ${conference} newsletter signup link`, function(done) {
+            const newsletterSelector = '.hero a[href*="newsletter"]';
+            const expectedText = 'Get Updates by Email';
+
+            const driver = new webdriver.Builder()
+                .forBrowser('firefox')
+                .build();
+
+            driver.get(url).then(function() {
+                driver.findElement(webdriver.By.css(newsletterSelector)).then(function(link) {
+                    link.getText().then(function(text) {
+                        assert.equal(text, expectedText);
+                    });
+                });
+                done();
+            });
+        });
+    });
+});
+
+test.describe('The Code of Conduct page...', function() {
+    this.timeout(timeOut);
+    const url = `${baseURL}/code-of-conduct`;
+    const expectedTitle = 'View Source • Code of Conduct';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
```

Comment:
I don't think this test really does anything useful. It seems to be a pretty static page with little interaction. Would this be covered in the link check above already? Just asserting that it has a 200 response there should be enough I think?